src/install/memcached/default-requirements.mdx

```diff
@@ -0,0 +1,6 @@
+---
+headingText: Check the compatibility and requirements
+componentType: default
+---
+
+Our integration is compatible with Memcached 1.4 and higher.
```

Comment:
i think style guide has us do "or higher", which is niche knowledge i know from EOL comms .

also wondering if we should beef it up by including the infra agent version they need? might be redundant so am just riffing

Reply:
poking around their repo, they also say there's no limitations on MacOS versions. you could add that, too.

they have different instructions on their repo for GoLang, tho. wanted to just call that out in case we need to update some instructions: https://github.com/newrelic/nri-memcached?tab=readme-ov-file#building